x-pack/plugins/alerting/server/saved_objects/migrations/index.test.ts

```diff
@@ -2606,6 +2606,21 @@ describe('successful migrations', () => {
         expect(migratedAlert870.references).toEqual([]);
       });
     });
+
+    test('migrates last run outcome order', () => {
+      const migration870 = getMigrations(encryptedSavedObjectsSetup, {}, isPreconfigured)['8.7.0'];
+      const rule = getMockData({
+        lastRun: {
+          outcome: 'failed',
+        },
+      });
+      const migratedAlert870 = migration870(rule, migrationContext);
+
+      expect(migratedAlert870.attributes.lastRun).toEqual({
+        outcome: 'failed',
+        outcomeOrder: 20,
+      });
```

Comment:
Nit: could have 3 test cases for the 3 values of the outcome

Reply:
yes, please add the other two cases

Reply:
Added more cases